pkgs/by-name/sp/spiffe-vault/package.nix

```
@@ -0,0 +1,61 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "spiffe-vault";
+  version = "0.6.2";
+
+  src = fetchFromGitHub {
+    owner = "philips-labs";
+    repo = "spiffe-vault";
+    rev = "v${finalAttrs.version}";
```

```suggestion
    tag = "v${finalAttrs.version}";
```

`tag` is usually preferred over `rev` to avoid fetching a branch of the same name instead of an actual release version tag